service/Gemfile.lock

```diff
@@ -1,17 +1,16 @@
 PATH
   remote: .
   specs:
-    agama-yast (12)
+    agama-yast (13)
       cfa (~> 1.0.2)
       cfa_grub2 (~> 2.0.0)
       cheetah (~> 1.0.0)
-      csv (~> 3.3)
+      csv (~> 3.2)
```

Comment:
Are we relaxing some of the versions here? (cvs, logger, etc.)

Reply:
Yes.

Reply:
I see this is a consequence of https://github.com/agama-project/agama/commit/3530726e6b539e8372af228cedd47ef78ef91699 so fully expected. Thanks for the pointer.

Reply:
It is related to https://github.com/agama-project/agama/commit/3530726e6b539e8372af228cedd47ef78ef91699.